account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -83,7 +80,7 @@ def filter_lines(self, lines):
         """
         self.ensure_one()
         payment_lines = self.env['payment.line'].\
-            search([('order_id.state', 'in', ('draft', 'open')),
+            search([('order_id.state', '!=', 'cancel'),
                     ('move_line_id', 'in', lines.ids)])
```

Comment:
The state on payment.lines is not the same as the payment order. It is called "Communication Type" and can have values `normal` or `structured`. I don't think we can use this field. But maybe add another field like `transaction_state` to capture whether the transaction is `pending`, `successful` or has `failed` and use that for the filter ?